apps/frontend/src/pages/auth/auth.tsx

```diff
@@ -1,20 +1,23 @@
 import { AppRoute } from "~/libs/enums/enums.js";
 import {
 	useAppDispatch,
-	useAppSelector,
+	// useAppSelector,
 	useCallback,
 	useLocation,
 } from "~/libs/hooks/hooks.js";
 import { actions as authActions } from "~/modules/auth/auth.js";
 import { type UserSignUpRequestDto } from "~/modules/users/users.js";
+import authIlustrationLeft from "~/pages/auth/assets/img/auth-ilustration-left.svg";
+import authIlustrationRight from "~/pages/auth/assets/img/auth-ilustration-right.svg";
 
+import "./assets/css/auth.css";
 import { SignInForm, SignUpForm } from "./components/components.js";
 
 const Auth: React.FC = () => {
 	const dispatch = useAppDispatch();
-	const { dataStatus } = useAppSelector(({ auth }) => ({
-		dataStatus: auth.dataStatus,
-	}));
+	// const { dataStatus } = useAppSelector(({ auth }) => ({
+	// 	dataStatus: auth.dataStatus,
+	// }));
```

Comment:
B34. There are no files, modules and parts of code that are not used in the project code, including commented code pats.

Reply:
yeah you're right, i forget about this